types/node-forge/index.d.ts

```diff
@@ -295,4 +295,16 @@ declare module "node-forge" {
             function create(): MessageDigest;
         }
     }
+
+    namespace cipher {
+        function createCipher(algorithm: string, payload: util.ByteBuffer): ForgeCipher;
+        function createDecipher(algorithm: string, payload: util.ByteBuffer): ForgeCipher;
+
+        interface ForgeCipher {
```

Comment:
Shouldn't this be `BlockCipher` instead? Also, could you please add tests for the new definitions?